<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewInformationDialog</class>
 <widget class="QDialog" name="ViewInformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View Information</string>
  </property>
  <widget class="QTextEdit" name="ContentsTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="ReportButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>530</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
